CONTRIBUTING.md

```diff
@@ -689,7 +691,9 @@ Ask them nicely whether they still intend to review your PR and find yourself an
 
 ### How can I get a committer to look at my PR?
 
-- Simply wait. Reviewers frequently browse open PRs and may happen to run across yours and take a look.
+- Improve skimmability: use a simple descriptive PR title (details go in commit titles) outlining _what_ is done, possibly also _why_.
```

Comment:
```suggestion
- Improve skimmability: use a simple descriptive PR title (details go in commit titles) outlining _what_ is done and _why_.
```

The why is the most important thing of any communication that isn't the code itself. If the the diff is huge, a "what" is helpful, and most changes should be small anyway in order to be reviewable.